docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+ethot:
+  image: counterstrafe/ethot
+  restart: always
+  links:
+    - "mysql:mysql"
+  environment:
+    MYSQL_HOST: 'mysql'
+    MYSQL_PORT: '3306'
+    MYSQL_DB: 'ebotv3'
+    MYSQL_USER: 'ebotv3'
+    MYSQL_PASS: 'ebotv3'
```

Comment:
We should also add things like `TOORNAMENT_API_KEY`, `EBOT_URL`, `EBOT_ADMIN_USER`, `EBOT_ADMIN_PASS`, etc.

Reply:
just a note for history: finding a better solution for secrets